apis/core/v1beta1/function_types.go

```diff
@@ -20,7 +20,7 @@ import (
 	componentsv1alpha1 "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	k8sgatewayapiv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+	k8sgatewayapiv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
```

Comment:
Old versions of the API ( v1beta1 ) don't need updating anymore?

Reply:
To use gateway API v1beta1, what's the minimum version of contour we should use?

Reply:
Covers using version v1beta1 of the Gateway API, with Contour v1.22.0 or higher.

https://projectcontour.io/guides/gateway-api/